src/index.ts

```diff
@@ -226,278 +209,244 @@ export const createInstance = (config: Config): Instance => {
       return tx;
     },
     zauction: {
-      needsToApproveZAuctionToSpendTokens: async (
+      getPaymentTokenInfo: async (
+        paymentTokenAddress: string,
+        chainName: string
```

Comment:
Could you infer the chain name based on how the instance was configured? So it's not a requirement to pass it in here?